frontend/src/components/ForecastPlot.vue

```diff
@@ -467,6 +471,23 @@ const downCSV = async () => {
   let filename = getFileName('csv')
   await downloadBlob(blob, filename)
   downloading.value.csv = false
+
+  trackEvent('Download Forecast CSV Button Clicked', {
+    reach_id: reach_id.value,
+    reach_name: reach_name.value
+  })
+}
+
+const trackEvent = (eventName, eventData = {}) => {
```

Comment:
I don't love that we have defined 
`const trackEvent = (eventName, eventData = {}) => {`
in multiple different components.

It's fine, but please consider whether it's possible to define a single trackEvent function that all the different components can import and use.

Reply:
you don't have to make this change, just a suggestion to think about